source/ht_inireader.cpp

```diff
@@ -62,6 +62,39 @@ namespace Hatchit {
             return key;
         }
 
+        char* INIReader::StreamReader(char* str, int len, void* stream)
```

Comment:
Fair enough.  I'm tempted to say that we should just use private statics over local statics anyway.